docs/validators/setup/mainnet/run-combined.md

```diff
@@ -214,7 +214,7 @@ For more information, see [Generate keys](../generate-keys.md).
    LISTENERS__RONIN__STATS__SECRET=WSyDMrhRBe111
    ```
 
-5. (Optional) Download the snapshot from the [ronin-snapshot](https://github.com/axieinfinity/ronin-snapshot) repo:
+5. (Optional) Download the snapshot from the [ronin-snapshot](https://github.com/axieinfinity/ronin-snapshot) repo (Prefer using Pebbledb):
```

Comment:
@huyngopt1994 Let's rewrite a bit to remove the parentheses:
(Optional) Download the snapshot from the [ronin-snapshot](https://github.com/axieinfinity/ronin-snapshot) repo, preferably using PebbleDB: